documentatie/softwarearchitectuur/overwegingen-talen-en-frameworks.md

```diff
@@ -146,7 +146,7 @@ Daarmee is de keuze voor Rust op basis van alle redelijke inschattingen en overw
 
 We hebben gekozen voor de volgende libraries:
 
-- De library [SQLite](https://www.sqlite.org/) die een SQL database engine implementeert. Dit is zeer populair, lichtgewicht en makkelijk in gebruik. Je hoeft bovendien niets te installeren. Het is handiger dan PostgreSQL, dat je zou moeten installeren op de computer of zou moeten meeleveren in de installer.
+- De library [SQLite](https://www.sqlite.org/) die een SQL-database engine implementeert. Dit is zeer populair, lichtgewicht en makkelijk in gebruik. Je hoeft bovendien niets te installeren. Het is handiger dan PostgreSQL, dat je zou moeten installeren op de computer of zou moeten meeleveren in de installer.
```

Comment:
Don't think the added hyphen is correct here

Reply:
I can remove it, jetbrains checker wanted it like this, but not sure if it checked for Dutch or English in this particular instance.

Reply:
Removed in 98484fe196db12ca50abccf69ffcf7277c3f24f0